pkg/reconciler/pipelinerun/resources/input_output_steps.go

```diff
@@ -24,15 +24,15 @@ import (
 )
 
 // GetOutputSteps will add the correct `path` to the output resources for pt
-func GetOutputSteps(outputs map[string]*resourcev1alpha1.PipelineResource, taskName, storageBasePath string) []v1beta1.TaskResourceBinding {
+func GetOutputSteps(outputs map[string]*resourcev1alpha1.PipelineResource, taskName string) []v1beta1.TaskResourceBinding {
 	var taskOutputResources []v1beta1.TaskResourceBinding
 
 	for name, outputResource := range outputs {
 		taskOutputResource := v1beta1.TaskResourceBinding{
 			PipelineResourceBinding: v1beta1.PipelineResourceBinding{
 				Name: name,
 			},
-			Paths: []string{filepath.Join(storageBasePath, taskName, name)},
+			Paths: []string{filepath.Join(taskName, name)},
```

Comment:
This doesn't look like it would produce a valid file path. Is this function still needed? It seems like git and storage are the only remaining output resources